includes/class-newspack-popups.php

```diff
@@ -640,6 +641,23 @@ public static function archive_campaign( $id, $status ) {
 		update_term_meta( $id, self::NEWSPACK_POPUPS_TAXONOMY_STATUS, $status ? 'archive' : '' );
 	}
 
+	/**
+	 * Prevent setting the default category when publishing.
+	 *
+	 * @param string $new_status New status.
+	 * @param string $old_status Old status.
+	 * @param bool   $post Post.
+	 */
+	public static function remove_default_category( $new_status, $old_status, $post ) {
+		if ( self::NEWSPACK_POPUPS_CPT === $post->post_type && 'publish' === $new_status ) {
```

Comment:
Currently, if you apply the default category after a prompt has been published (the only way to apply it after this PR), it will be removed the next time the post is updated. 

This should prevent the action from running when a prompt is updated after already being published:

```suggestion
		if ( self::NEWSPACK_POPUPS_CPT === $post->post_type && 'publish' !== $old_status && 'publish' === $new_status ) {
```

Reply:
This would prevent default-category-unsetting when the prompt is re-activated, which might happen often.

Reply:
That's not what I see when I test. How can I replicate? Here's what I tested:

1. Create a new prompt and publish without setting any categories: the prompt is activated without the default category.
2. Deactivate the prompt.
3. Assign the default category (only).
4. Reactivate the prompt—the default category is removed.